neqo-common/src/qlog.rs

```diff
@@ -48,6 +54,11 @@ impl NeqoQlog {
         })
     }
 
+    #[must_use]
+    pub fn inner(&self) -> Rc<RefCell<Option<NeqoQlogShared>>> {
+        Rc::clone(&self.inner)
```

Comment:
```suggestion
        self.inner.clone()
```

Compiles for me on Rust `v0.175.0`. Maybe a bit easier to read?

Reply:
The guidance for Rc::clone is to use this form: https://rust-lang.github.io/rust-clippy/master/index.html#/clone_on_ref_ptr

Reply:
Oh, I did not know. Thanks for the pointer!